<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>285</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutTimer">
          <item>
           <widget class="QLabel" name="labelTimer">
            <property name="text">
             <string>Timer (s):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderTimer">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberTimer">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutDisConnect">
          <item>
           <widget class="QPushButton" name="pushButtonConnect">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDisconnect">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutMax">
          <item>
           <widget class="QLabel" name="labelMax">
            <property name="text">
             <string>Max:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderMax">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberMax">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>41</height>
             </size>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutSTimer">
          <item>
           <widget class="QPushButton" name="pushButtonStartTimer">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStopTimer">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutMin">
          <item>
           <widget class="QLabel" name="labelMin">
            <property name="text">
             <string>Min:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderMin">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberMin">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayoutIP">
          <item>
           <widget class="QLabel" name="labelIP">
            <property name="text">
             <string>IP:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditIP">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelStatus">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Status: &lt;font color=&quot;red&quot;&gt;Desconectado&lt;/red&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="textBrowserDados">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>285</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProduto_de_Dados">
    <property name="title">
     <string>Produto de Dados</string>
    </property>
   </widget>
   <addaction name="menuProduto_de_Dados"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderTimer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberTimer</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
